### ISSUE TITLE
Desktop "Show Hidden" setting not preserved upon restarting

### ISSUE THREAD

**Issue by isf63** [lxqt/pcmanfm-qt#1828]
##### Expected Behavior
The 'Show Hidden' option when checked, remains checked when restarting PCManFM-Qt's desktop. 

##### Current Behavior
It is always set to unchecked upon restarting the processes.

##### Steps to Reproduce (for bugs)
1. Check 'Show Hidden' in the desktop's right-click menu.
2. Add some dot files do the desktop folder. They should appear (optionally greyed).
3. Open lxqt-config-session and Stop and Start the Desktop module.

##### Context
Hopefully this is reproducible.

There may be other desktop settings not preserved upon relaunching. 

##### System Information

* PCManFM-Qt version: git latest
* Qt Version: 5.15.11
* liblxqt Version: git latest

**Comment by tsujan** [comment]
Yes, I've known about it for a long time but waited to see if any user would miss it or not. I had a plan to remove its key if no one wanted it. After years, you're the first one ;)

Purely logical reasons aside, I still doubt whether it's really useful to show hidden files on Desktop on starting it, but the code supports it, independently of the main window.

**Comment by isf63** [comment]
I keep symlinks to the XDG user directories on the desktop for quick access. I also added symlinks to the standard $HOME hidden directories, and noticed upon restarting the Desktop module for another reason that they disappeared. Here's a picture showing my usage:

![desktop-screenshot](https://github.com/lxqt/pcmanfm-qt/assets/121320947/a91ecaca-4cb9-42ab-81bf-31703165bd8e)

**Comment by tsujan** [comment]
Done in https://github.com/lxqt/pcmanfm-qt/pull/1922. And sorry for the long delay for a simple change; I was busy with many other codes.

**Comment by isf63** [comment]
Thanks a lot! There are only a few use cases, but in my case it's useful to have hidden files shown. Fortunately most of the infrastructure was already in place.

**Comment by tsujan** [comment]
> in my case it's useful to have hidden files shown.

It was a bug, technically speaking. But only you and I noticed it — and I remained silent ;)
